packages/server/lib/browsers/electron.js

```diff
@@ -355,11 +355,8 @@ module.exports = {
   },
 
   _listenToOnBeforeHeaders (win) {
-    const isFirstLevelIFrame = (frame) => {
-      if (frame.parent && !frame.parent.parent) return true
-
-      return false
-    }
+    // true if the frame only has a single parent, false otherwise
+    const isFirstLevelIFrame = (frame) => (!!frame?.parent && !frame.parent.parent)
```

Comment:
Does this code work in Node 12 since [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility) was introduced in Node 14? I think we are still [supporting](https://docs.cypress.io/guides/getting-started/installing-cypress#Node-js) 12 even though 12 maintaince ends in either [May or June](https://nodejs.org/en/about/releases/)?
```suggestion
    const isFirstLevelIFrame = (frame) => (frame && frame.parent && !frame.parent.parent)
```

Reply:
After team discussion, it sounds like the server is run with whatever version of node electron is bundled with, which should be node 16 with electron 15